src/styles/blocks/stars.css

```diff
@@ -0,0 +1,40 @@
+.stars {
+  display: flex;
+  margin-bottom: 24px;
+}
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  height: 16px;
+  width: 16px;
+
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars__review {
+  width: 53px;
+  height: 16px;
+
```

Comment:
a fixed "width" might not be necessary if proper paddings or margins are arranged for this particular element.

Reply:
agree